frontend/src/components/TaskCard.tsx

```diff
@@ -0,0 +1,172 @@
+import * as React from 'react'
+import styled from 'styled-components'
+import Card from './Card'
+import { Header, Subheader, Paragraph } from './Typography'
+import Button from './Button'
+
+const ExpandedTaskCardWrapper = styled.div`
```

Comment:
I noticed when you expand a task card, it lengths the whole page (you can scroll up and down). Can you make it so this doesn't happen? The expanded task card should push down other cards below it but not lengthen the page